web/src/people.ts

```diff
@@ -31,25 +31,25 @@ export type User = {
     // used for caching result of remove_diacritics.
     name_with_diacritics_removed?: string | undefined;
     date_joined: string;
-    is_active: boolean;
+    is_active?: boolean;
     is_owner: boolean;
     is_admin: boolean;
     is_guest: boolean;
-    is_moderator: boolean;
-    is_billing_admin: boolean;
+    is_moderator?: boolean;
+    is_billing_admin?: boolean;
```

Comment:
Hmm, under what circumstances are these `is_` fields optional? Kinda smells like it could be not intended.

Reply:
Hmm, in fact, `is_moderator` does not exist at all.

https://github.com/zulip/zulip/blob/ce71e5ad8c356edf5d5c7024148d986254816eda/zerver/lib/users.py#L451-L471

`is_billing_admin` and `timezone` are censored for spectators.

https://github.com/zulip/zulip/blob/ce71e5ad8c356edf5d5c7024148d986254816eda/zerver/lib/users.py#L516-L523